jaraco/abode/socketio.py

```diff
@@ -202,8 +203,9 @@ def _step(self, intervals):
                 intervals.reset()
 
             name = event.__class__.__name__.lower()
-            method = getattr(self, f'_on_websocket_{name}')
-            method(event)
+            with suppress(AttributeError):
+                method = getattr(self, f'_on_websocket_{name}')
+                method(event)
```

Comment:
The AttributeError was introduced as part of da40793fcd2ffe31d31c59ecea2979129b17b9aa. It wasn't obvious to me if there were methods silently ignored by the lack of an `else:` clause. Now we know. Glad to git this fixed.